backend/src/categories/categories.module.ts

```diff
@@ -18,7 +18,7 @@ import { UserService } from 'src/user/user.service';
       secret: 'secret',
       signOptions: { expiresIn: '1d' },
     }),
-    TypeOrmModule.forFeature([Category, Words, Choices, User]),
+    TypeOrmModule.forFeature([Category, Words, Choices,User]),
   ],
```

Comment:
i think code still formatted. can you setup prettier.

https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code#step-2-formatting-code-on-save